src/routes/categories/components/CategoryList.js

```diff
@@ -7,19 +7,33 @@ import Caption from '../../../modules/common/components/Caption'
 import CategoryListItem from './CategoryListItem'
 import CategoryModel from '../../../modules/endpoint/models/CategoryModel'
 import styled from 'styled-components'
+import iconPlaceholder from '../assets/IconPlaceholder.svg'
 
 const List = styled.div`
   & a {
     ${props => props.theme.helpers.removeLinkHighlighting}
   }
 `
 
+const Centering = styled.div`
```

Comment:
This can be added to CategoryIcon instead of Centering

Reply:
how?

Reply:
I meant the `text-align: center;` I think it can be moved to CateoryIcon.

Do we really need the text-align here? I mean there is no text in CategoryIcon.

Reply:
text-align centers all inline elements inside its container such as img